ext/nnstreamer/tensor_decoder/meson.build

```diff
@@ -1,6 +1,7 @@
 # direct video
 decoder_sub_direct_video_sources = [
-  'tensordec-directvideo.c'
+  'tensordec-directvideo.c',
+  'tensordecutil.c'
```

Comment:
If tensordecutil.o commonly linked to others, how about ```declare_dependency`` for it?

Reply:
Another PR in next branch. Please see the rebased.